source/CDebug.h

```diff
@@ -18,6 +18,8 @@ class CDebug
 #else
         CLEO::Trace(CLEO::eLogLevel::Default, "CLEO v%s", CLEO_VERSION_STR);
 #endif
+
+        std::remove("cleo.log"); // delete CLEO 4 log file if present
```

Comment:
not sure we should do it. CLEO5 does not own this file and we should not delete anything user did not ask for

Reply:
Every time I ask somebody for log file they send old one.
Cleo logs were always truncated on new game start, so it is not like someone could store there some precious data.

Reply:
You just assume if there is a file named "cleo.log" it is a file created by CLEO4. This is just an assumption and we should not delete user files just because someone made a mistake.